Selectors/src/main/java/io/doist/recyclerviewext/choice_modes/Selector.java

```diff
@@ -50,16 +50,56 @@ public void toggleSelected(long id) {
 
     public abstract void clearSelected();
 
+    @Deprecated
```

Comment:
We can add a comment saying something like:
```
  /**
   * @deprecated use [addOnSelectionChangedListener] to add a listener (equivalent to setOnSelectionChangedListener(validListener)) or [removeAllUpdateListeners] to remove all listeners (equivalent to setOnSelectionChangedListener(validListener)).
   */
```

Also, why are we marking this as deprecated instead of just removing the method? 🤔

Reply:
It's a public library, shouldn't we keep it deprecated for while? 🤔

Reply:
+1 on adding a comment, but I don't think it should be _so_ detailed.